### PR TITLE
Adding a describe snapshots step

### PR DESCRIPTION
**Pull request by woodburndesigns**

### CODE REVIEW

File: steps/snapshots-describe/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+FROM python:3-alpine
+RUN apk --no-cache add bash ca-certificates curl git jq openssh && update-ca-certificates
+RUN pip --no-cache-dir install "https://packages.nebula.puppet.net/sdk/support/python/v1/nebula_sdk-1-py3-none-any.whl"
+RUN pip --no-cache-dir install boto3
+COPY "./step.py" "/nebula/step-ec2-describe-instances.py"
+ENTRYPOINT []
+CMD ["python3", "/nebula/step-ec2-describe-instances.py"]
+
+LABEL "org.opencontainers.image.title"="Describe AWS EC2 instances"
```
Comment (kenazk) [inline steps/snapshots-describe/Dockerfile]: Update to snapshot

File: steps/snapshots-describe/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# aws-ec2-step-instances-describe
```
Comment (kenazk) [inline steps/snapshots-describe/README.md]: needs update

File: steps/snapshots-describe/Dockerfile (hunk 1 above)
Comment (woodburndesigns) [reply steps/snapshots-describe/Dockerfile]: Good catch.

**Comment by woodburndesigns** [comment]
Closing in favor of: https://github.com/relay-integrations/relay-aws-ebs/pull/4

**Pull request closed without merging**